examples/remix-vite-babel/app/root.tsx

```diff
@@ -0,0 +1,62 @@
+import { ActionFunctionArgs, json, LoaderFunctionArgs } from "@remix-run/node";
+import {
+  Links,
+  Meta,
+  Outlet,
+  Scripts,
+  ScrollRestoration,
+} from "@remix-run/react";
+import { linguiServer, localeCookie } from "./modules/lingui/lingui.server";
+import { useLocale } from "./modules/lingui/lingui";
+
+export async function action({ request }: ActionFunctionArgs) {
+  const formData = await request.formData();
+
+  const locale = formData.get('locale') ?? 'en'
+
+  return json(
+    null,
+    { headers: {
+      'Set-Cookie': await localeCookie.serialize(locale)
+    } },
+  );
+}
+
+export async function loader({ request }: LoaderFunctionArgs) {
+  const locale = await linguiServer.getLocale(request);
+
+  return json(
```

Comment:
I wonder if an example of setting a title from the loader would be a good use case?

Reply:
Good idea. I added this in the example, thanks for the suggestion